api/src/pcapi/connectors/thumb_storage.py

```diff
@@ -11,8 +11,9 @@ def create_thumb(
     image_index: int,
     crop_params: tuple = None,
     ratio: float = IMAGE_RATIO_PORTRAIT_DEFAULT,
+    keep_ratio: bool = False,
```

Comment:
l'argument a une valeur par défaut

est-ce qu'il faudrait transformer son type en ?

```suggestion
    keep_ratio: Optional[bool] = False,
```

Reply:
Non, `Optional` c'est quand `None` est une valeur possible.

> typing.Optional
> 
> Optional type.
> 
> Optional[X] is equivalent to X | None (or Union[X, None]).
> 
> Note that this is not the same concept as an optional argument, which is one that has a default.
> An optional argument with a default does not require the Optional qualifier on its type annotation just
> because it is optional

[source](https://docs.python.org/3/library/typing.html)